src/components/OverlayAndCancelButton/OverlayAndCancelButton.tsx

```diff
@@ -23,3 +23,32 @@ export function LoadingOverlayAndCancelButton(loading: boolean, pid: string) {
         </>
     );
 }
+
+//Overlay to successfully terminate processes for tools using pkexec
+export function LoadingOverlayAndCancelButtonPkexec(
+    loading: boolean,
+    pid: string,
```

Comment:
There is an issue in your code. You have disallowed the possibility for this pid to be null then later have a conditional for null. This means that there exists an potential for pid to be null and introduces a potential unhandled exception. 

- [ ] Please introduce some exception handling to cater for this case.

Reply:
Added exception handling and updated goldeneye for testing